tests/tests.go

```diff
@@ -209,6 +209,22 @@ func initKubemacpoolParams(rangeStart, rangeEnd string) error {
 	return nil
 }
 
+func getVmFailCleanupWaitTime() time.Duration {
+	managerDeployment, err := testClient.KubeClient.AppsV1().Deployments(managerNamespace).Get(names.MANAGER_DEPLOYMENT, metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred(), "Should succeed getting Deployment object")
+	Expect(len(managerDeployment.Spec.Template.Spec.Containers)).To(Equal(1))
+
+	for _, arg := range managerDeployment.Spec.Template.Spec.Containers[0].Args {
+		waitTime := 0
+		if elementsParsed, err := fmt.Sscanf(arg, "--wait-time=%d", &waitTime); elementsParsed == 1 {
+			Expect(err).ToNot(HaveOccurred(), "Should succeed parsing wait-time arg")
+			return time.Duration(waitTime) * time.Second
+		}
+	}
+
+	return 0
```

Comment:
I'm not sure if I should return an Error if this argument is not there. what do you think?

Reply:
I think we should return an error with a message "could not get ..."

Reply:
ack. I agree